platform/android/MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -45,11 +45,9 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled = true
-            minifyEnabled false
```

Comment:
@tobrun Why does this line need to be removed? Isn't it better to leave the explicit setting?